katpoint/timestamp.py

```diff
@@ -138,10 +138,7 @@ def __iadd__(self, other):
 
     def __rsub__(self, other):
         """Subtract timestamp from seconds (or another timestamp) and return result."""
```

Comment:
Sorry to nitpick, but the docstring still has to change to reflect that this can only result in an interval.

Reply:
i have just committed a change which i believe addresses your request sufficiently well. please either make an explicit suggestion if further change is required, or make the change yourself.